Dockerfile

```diff
@@ -21,4 +21,9 @@ ENV DATABASE_CONFIG=/var/lib/drone/drone.sqlite
 
 ADD drone_static /drone_static
 
+# Alpine Linux doesn't use pam, which means that there is no /etc/nsswitch.conf,
```

Comment:
Go's net package also parses nsswitch.conf file by itself so it might help even if pam isn't installed.

IIRC the situation that did not work was that the hosts-file wasnt looked at all which docker used to link containers togeher earlier. Since Docker has moved to using an DNS server for the networks the nsswitch.conf might not be required anymore. Idk.